docs/source/server/concurrency.rst

```diff
@@ -0,0 +1,43 @@
+Multi-Tenancy & Concurrency
+==========================
+
+Graphistry `register()` and `plot()` are stateful. Methods are the concurreny boundary.
+
+This is because on `plot`, the `ArrowUploader` manages user auth token refresh on session
+state. 
+
+`Plottable._pygraphistry.session` = `Graphistry.session`
+
+A `Plottable`'s state is tied to the client used to create it.
+
+Example
+-------
+
+.. code-block:: python
```

Comment:
While verbose, it's helpful to do examples that we see here frequently so we can just send docs links when people ask (or search for those keywords) --

* Multi-tenant (this example)
* Multiple servers (staging vs prod)
* Multi-threaded

Reply:
done